CMakeLists.txt

```diff
@@ -52,6 +52,7 @@ set(NOVELRT_VULKAN_VERSION "1.2.172" CACHE STRING "Vulkan version")
 set(NOVELRT_VULKAN_MEMORY_ALLOCATOR_VERSION "2.3.0" CACHE STRING "VulkanMemoryAllocator version")
 
 if (NOVELRT_INCLUDE_CONAN_PATHS)
+  set(CONAN_CMAKE_SILENT_OUTPUT ON)
```

Comment:
What's this do?

Reply:
This calms down the spewing of Conan information during CMake generation to a measily ~17 lines.